code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -44,7 +44,25 @@
 		else
 			return 0
 
+/obj/item/weapon/melee/baton/proc/update_process()
+	if(status)
+		SSobj.processing.Add(src)
+	else if(!bcell)
+		SSobj.processing.Remove(src)
+	else

```

Comment:
uh, this else and the else if above it both end up running the same code. so you shouldn't need the else if?